src/styles/page-template/_template.njk

```diff
@@ -124,7 +124,18 @@
                     {% endblock %}
                     {% block pageContent %}
                         <div class="page__container container {{ containerClasses }}">
-                            {% block preMain %}{% endblock %}
+                            {% block preMain %}
+                                {% if pageConfig.breadcrumbs is defined and pageConfig.breadcrumbs %}
+                                    {% from "components/breadcrumb/_macro.njk" import onsBreadcrumb %}
+                                    {{
+                                        onsBreadcrumb({
+                                            "id": pageConfig.breadcrumbs.id,
+                                            "ariaLabel": pageConfig.breadcrumbs.ariaLabel,
+                                            "itemsList": pageConfig.breadcrumbs.itemsList
+                                        })
+                                    }}
+                                {% endif %}
+                            {% endblock %}
```

Comment:
This looks good but I think using the existing block `preMain` may cause issues. If other services use this block for other stuff (not even sure that they do) it would replace this breadcrumb code. Not sure if needs to even sit in a `block`. Could be just above `preMain`

Reply:
Oh yeah good point. Ill move it